src/main.js

```diff
@@ -189,27 +193,43 @@ function goBackToMainSaved(){
 
 //Iteration 2
 function showUserPoster(){
-  // event.preventDefault()
-  // var newUserImage = inputFieldForImage.value;
-  // var newUserTitle = inputFieldForTitle.value;
-  // var newUserQuote = inputFieldForQuote.value;
-  // var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
-  // image.src = newUserImage;
-  // title.innerText = newUserTitle;
-  // quote.innerText = newUserQuote;
-  // images.push(newUserImage);
-  // titles.push(newUserTitle);
-  // quotes.push(newUserQuote);
-  // goBackToMainMake();
+  event.preventDefault()
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
+  image.src = newUserImage;
+  title.innerText = newUserTitle;
+  quote.innerText = newUserQuote;
+  images.push(newUserImage);
+  titles.push(newUserTitle);
+  quotes.push(newUserQuote);
+  goBackToMainMake();
 }
 // Iteration 3
-function pushUserPosterToArray(){
+function saveUserPoster(){
+  var newUserImage = inputFieldForImage.value;
+  var newUserTitle = inputFieldForTitle.value;
+  var newUserQuote = inputFieldForQuote.value;
+  var newUserPoster = new Poster(newUserImage, newUserTitle, newUserQuote);
+
+  savedPosters.push({newUserPoster})
```

Comment:
awesome!
instantiating a new poster class and pushing the entire object with it's properties, by instatiating the poster class and pushing it into the savedPosters array.